docs/whitepapers/kalmanfilter/kalmanfilter.tex

```diff
@@ -0,0 +1,345 @@
+\\
+
+\noindent
+Additionally, the drone's vertical velocity \( v_{z_t} \) and vertical acceleration \( a_{z_t} \) are defined by the rate of altitude change \( \dot{z}_t \) and the rate of vertical acceleration change \( \dot{a}^z_{T_t} - g \), respectively. The term \( \dot{a}^z_{T_t} \) is derived from the quadcopter's upward thrust and serves as the system's input, while \( g \) is considered a constant input in the opposite direction.
+\\
+
+\noindent
+Let \( y_{t}^z = z_t \) be the output equation of the system. Then the corresponding state-space representation is,
+
+\begin{equation}
+\begin{bmatrix}
+v_{v,t}\\
+a_{t}^z
+\end{bmatrix} =
+\begin{bmatrix}
+0 & 1 \\
+0 & 0 
+\end{bmatrix}
+\begin{bmatrix}
+z\\
+v_{v,t}
+\end{bmatrix} + 
+\begin{bmatrix}
+0 & 0 \\
+1 & 1 \\ 
+\end{bmatrix}
+\begin{bmatrix}
+a_{T,t}^z \\ 
+-g
+\end{bmatrix},
+\end{equation}
+\\
+
+\noindent
+for the Barometer sensor's output, denoted by \( y_{barom} \), is described by the equation:
+\begin{equation}
+y^{barom} = z + bias + V^{barom}
+\end{equation}
+\\
+where, 
+\begin{equation}
+bias_{t+1} = bias_t + W_{t}^{bias}
+\end{equation}
+the bias should stay consistent throughout readings 
+\\
+\\ 
+
+
+\noindent
+for the GPS sensor's output, denoted by \( y_{gps} \), is described by the equation:
+\begin{equation}
+y_{gps} = z + V^{gps}
+\end{equation}
+
+\begin{equation}
+y_{gps} =
+\begin{bmatrix}
+    1 & 0
+\end{bmatrix}
+x + V^{gps}
+\end{equation}
+\\
+
+\noindent
+The Time-of-Flight (ToF) sensor's output, denoted by \( y_{ToF} \), is described by the equation:
+
+\begin{equation}
+y_{ToF} = z + V^{ToF}
+\end{equation}
+\\
+
+\noindent
+throught all sensors \( y \) represents the output from the sensor. The variable \( z \) signifies the quadcopter's altitude, which is the measurement for all sensors. The term \( V \) encapsulates the measurement noise or errors associated with the sensors. This noise term, \( V \), encompasses various factors such as sensor inaccuracies, the impact of environmental conditions on sensor performance, and any systematic bias that might be inherent in the sensor's readings.
+\\
+
+\noindent
+Equation (2) describes the continuous-time altitude dynamics of the quadcopter. The dis-
+cretization of the altitude dynamics of the system (2) with a sampling frequency of Ts using the zero-order hold technique is,
+
+\begin{equation}
+\begin{bmatrix}
+z_{t+1}\\
+v_{z,t+1}
+\end{bmatrix} =
+\begin{bmatrix}
+1 & T_s \\
+0 & 1 
+\end{bmatrix}
+\begin{bmatrix}
+z_t\\
+v_{s,t}
+\end{bmatrix} + 
+\begin{bmatrix}
+{1/2}{T^2}_s & {1/2}{T^2}_s \\
+T_s & T_s \\ 
+\end{bmatrix}
+\begin{bmatrix}
+a_{T,t}^z \\ 
+-g
+\end{bmatrix},
+\end{equation}
+\subsection{state vector definition}
```

Comment:
Does this have to be a section? The state vector should be a **vector**. This is a row vector.